src/Explorer/Explorer.tsx

```diff
@@ -654,6 +696,13 @@ export default class Explorer {
     if (!notebookContentItem || !notebookContentItem.path) {
       throw new Error(`Invalid notebookContentItem: ${notebookContentItem}`);
     }
+    if (
+      notebookContentItem.type === NotebookContentItemType.Notebook &&
+      userContext.features.notebooksTemporarilyDown === false &&
+      userContext.features.phoenix === true
+    ) {
+      this.allocateContainer();
```

Comment:
this is a temporary place holder till we have reset API right?

Reply:
On click of notebook making allocate call

Reply:
On click of notebook making allocate call